agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1510,6 +1516,11 @@ void ap_manager_thread::handle_hostapd_attached()
 {
     LOG(DEBUG) << "handling enabled hostapd";
 
+    ap_wlan_hal->read_supported_channels();
```

Comment:
Changed all misuse of the get_radio_info().supported_channels to now use the new preferred_channels
Also changed it so now the base_wlan_hal_dwpal's refresh_radio_info will save the supported channels and added a function called `update_preference_channels_from_supoorted_channels` what will copy the supported channels to the preferred_channels in case the acs is not enabled